src/Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <Nullable>enable</Nullable>
     <StrongNameKeyId>MicrosoftAspNetCore</StrongNameKeyId>
     <OutputType>Exe</OutputType>
+    <IsTestProject>false</IsTestProject>
```

Comment:
"Program does not contain a static 'Main' method suitable for an entry point"

https://dev.azure.com/dnceng/public/_build/results?buildId=1110881&view=logs&j=60adcdc4-a9f0-5668-20f7-b72181c68fb2&t=b1a542ef-636a-581a-98b6-aea5343cc67d&l=322